csvcubed/csvcubed/readers/cubeconfig/v1/columnschema.py

```diff
@@ -78,25 +92,66 @@ def map_to_new_qb_dimension(
         )
         # The NewQbCodeList and Concepts are populated in the NewQbDimension.from_data() call
         # the _get_code_list method overrides the code_list if required.
-        new_dimension.code_list = self._get_code_list(new_dimension, csv_column_title)
-        return new_dimension
+        (
+            new_dimension.code_list,
+            code_list_schema_validation_errors,
+        ) = self._get_code_list(
+            new_dimension,
+            csv_column_title,
+            cube_config_minor_version,
+            cube_config_path=config_path,
+        )
+        return (new_dimension, code_list_schema_validation_errors)
 
     def _get_code_list(
-        self, new_dimension: NewQbDimension, csv_column_title: str
-    ) -> Optional[QbCodeList]:
-
+        self,
+        new_dimension: NewQbDimension,
+        csv_column_title: str,
+        cube_config_minor_version: Optional[int],
+        cube_config_path: Optional[Path],
+    ) -> Tuple[Optional[QbCodeList], Optional[list[ValidationError]]]:
         if isinstance(self.code_list, str):
             if looks_like_uri(self.code_list):
-                return ExistingQbCodeList(self.code_list)
+                return (ExistingQbCodeList(self.code_list), None)
+            # The following elif is for cube config v1.1. This also requires the user to define the configuration in the build command, and therefore cube_config_path.
+            elif (
+                cube_config_minor_version
+                and cube_config_minor_version >= 1
+                and cube_config_path
+                and code_list_config_json_exists(
+                    Path(self.code_list), cube_config_path.parent
+                )
+            ):
+                code_list_config_path = (
```

Comment:
We should ideally be able to support web absolute file paths here in addition to relative paths. So could you check if the path is absolute, and if it is just use that path, else use `cube_config_path.parent / self.code_list`?